tests/Intl/Idn/IdnTest.php

```diff
@@ -76,7 +76,7 @@ public function getData()
         $tests = array();
 
         while (false !== ($line = fgets($h))) {
-            if ("\n" === $line || '#' === $line[0]) {
+            if (PHP_EOL === $line || '#' === $line[0]) {
```

Comment:
this looks wrong to me. If you don't alter the newlines when cloning the code, the file will *not* use Windows line endings and this check will break.

Reply:
well, In fact this change fix the tests. The line ends with `\r\n` in windows (tested on a Windows 10 laptop and Github Action)

```
while (false !== ($line = fgets($h))) {
  var_dump(json_encode($line));
```
linux: `string(20) ""# IdnaTestV2.txt\n""`
windows: `string(22) ""# IdnaTestV2.txt\r\n""`

I can add `if ("\n" === $line || "\r" === $line || "\r\n" === $line || '#' === $line[0]) {` if you are not confortable.

Reply:
maybe Github Actions configures git with `core.auto_crlf=true` while Appveyor does not.

But in any case, our tests should keep working if someone uses `core.auto_crlf=input` or `core.auto_crlf=false` (`core.auto_crlf=input` is IMO a much saner value on Windows as it makes you work on the original source code, which works fine in all Windows editor except old versions of Notepad.exe).

I would go with `"\n" === $line || PHP_EOL === $line`, or maybe `"\n" === $line || "\r\n" === $line`. no need to care about standalone `\r`. this is dead since at least a decade.

Reply:
good catch for `core.auto_crlf`.. Fixed